library/std/src/sys/pal/unix/thread.rs

```diff
@@ -220,23 +219,18 @@ impl Thread {
     #[cfg(target_os = "vxworks")]
     pub fn set_name(name: &CStr) {
         // FIXME(libc): adding real STATUS, ERROR type eventually.
-        weak! {
+        extern "C" {
             fn taskNameSet(
```

Comment:
should we add it to libc crate at some point ? maybe not 0.2 branch but for the next 0.3 release.